highfidelity/hifi
# Voxel paint

# QA Test
- Test this with multiple people.
- This QA requires HMD and handcontrollers
- Use this build
1. Test Lead: Run script https://rawgit.com/thoys/hifi/voxelPaint/unpublishedScripts/marketplace/voxelPaint/createVoxelPaint.js using `Edit-> Running Scripts -> From URL`
2. Test Lead: Click or trigger the arrows to initialize the voxelPaint platform and rezzer.
3. Each user: trigger the palette shaped rezzer to receive a palette and brush.
4. Each user: test if the different brushes and colors work, and if they're selectable.
5. Each user: stress test the rezzer, try rapidly triggering the palette rezzer, each exessive palette should delete within 20 seconds.
6. Each user: try un-equipping either the palette and tools, in both orders, and notice that the palette and tool both disappears when that is done.
7. Each user: try to lose tracking for the hand that is drawing, this should not make any glitchy lines that move towards the avatars hips. After regaining tracking, a re-trigger is required to start drawing again.
